packages/core/js-client/src/clientPeer/__test__/client.spec.ts

```diff
@@ -17,15 +17,44 @@
 import { JSONValue } from "@fluencelabs/interfaces";
 import { it, describe, expect } from "vitest";
 
+import { ExpirationError, SendError } from "../../jsPeer/errors.js";
 import { CallServiceData } from "../../jsServiceHost/interfaces.js";
-import { doNothing } from "../../jsServiceHost/serviceUtils.js";
 import { handleTimeout } from "../../particle/Particle.js";
 import { registerHandlersHelper, withClient } from "../../util/testUtils.js";
 import { checkConnection } from "../checkConnection.js";
 
 import { nodes, RELAY } from "./connection.js";
 
 describe("FluenceClient usage test suite", () => {
+  it("Should resolve at TTL when fire and forget behavior is used", async () => {
```

Comment:
How does fire and forget relates to TTL? I was under impression, that these are completely unrelated.

Reply:
Yeap, wrong description. Changed